### DISCUSSION TITLE
Welcome to Slated-Obsidian Discussions!

### DISCUSSION THREAD

**Discussion by tgrosinger** [tgrosinger/slated-obsidian#4] [General]
## 👋 Welcome!

Use this space to share ideas, ask questions, and help envision the future of task management within Obsidian.

Discussions are brand new, so this will be a bit of an experiment!

**Comment by ericaxu** [comment]
This is a cool beta feature! 👀

**Comment by rklasen** [comment]
The goal of this plugins looks fantastic. I can't really contribute other than give encouragement :smiley:

**Reply by tgrosinger** [reply]
Thanks @rklasen! I'll post in the plugins chat room on Discord when I am ready for beta testers.

**Comment by menagerie198** [comment]
Hi Tony. how do I stop a recurring reminder from popping up on each new daily page? I set a recurring reminder, but ticked the task off. It continues to  pop up each day.

**Reply by tgrosinger** [reply]
@menagerie198, I think I fixed that issue in a recent release. Please make sure you are running version 0.0.7. If the issue is still happening in 0.0.7 for you please [file a new issue](https://github.com/tgrosinger/slated-obsidian/issues/new/choose) so I can get it fixed right away! Thank you 🙂

**Reply by menagerie198** [reply]
All seems well now! Cheers

**Comment by adxsoft** [comment]
I have  situation where, for example, I complete a monthly task and once marked as complete I want the task to appear in a daily note 1 month in the future *regardless* of whether the daily note exists. 

In other words if the daily note (one month in the future) does not exist then 
- the Slated plugin should create the note automatically and post the task in the new note.
if the future daily note does exist then
- the task is added to the existing daily note 

So am I right in my understanding that completed a repeat task will only repeat if the future daily note exists?

Thanks
AD

**Reply by tgrosinger** [reply]
@adxsoft, Slated will create notes when needed for both repeating and moving tasks.
